apollo-environment/src/main/java/com/spotify/apollo/environment/ApolloEnvironmentModule.java

```diff
@@ -65,8 +65,14 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(ApolloEnvironmentModule.class);
 
-  public static ApolloEnvironmentModule create() {
```

Comment:
Reading the other commits, it looks like the idea is to always use `ClientDecoratorOrder` when creating `ApolloEnvironmentModule`.

I would still keep the default, argument-less constructor for simple use cases where the is no need to control the order.